src/app/app.module.ts

```diff
@@ -5,10 +5,15 @@ import { HttpClientModule } from '@angular/common/http';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { RouterModule } from '@angular/router';
+import { HomeComponent } from './_teacher/home/home.component';
+import { HomestudentsComponent } from './_students/homestudents/homestudents.component';
+import { FeedstudentsComponent } from './_students/feedstudents/feedstudents.component';
+import { StatsstudentsComponent } from './_students/statsstudents/statsstudents.component';
+import { NavbarstudentsComponent } from './_students/navbarstudents/navbarstudents.component';
 
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent, HomestudentsComponent, FeedstudentsComponent, StatsstudentsComponent, NavbarstudentsComponent],
```

Comment:
Formatting

Reply:
wdym ?

Reply:
the line is extra long, difficult to read on my computer/phone
I think there is an issue around formatting the file (prettier ?)

Reply:
it's extra-minor don't waste time on this

Reply:
100 % right gonna fix it right away